my-app/src/components/CommentItem.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from "styled-components";
+import {UserType} from "../pages/homePage/BlogHomeContext";
+
+type CommentType ={
+    _id?:string
+    comment:string
+    user:UserType
+}
+type CmtType = {
+    cmt:CommentType
+}
+export const CommentItem = ({cmt} :CmtType) => {

```

Comment:
why not `export const CommentItem = ({comment} : CommentType) => {` ?

Reply:
I pass the comment and also other info about the user, I use destruction a little bit, but I believe I need to pass an object from the ArticlePage component and then extract the comment and user name from it. Tried to pass only a comment but I need also the name of the user